### ISSUE TITLE
Not being able to load the examples

### ISSUE THREAD

**Issue by aitorllj93** [nextbigsoundinc/stylemark#8]
Hi,

I've been trying to load the examples inside the comments but the regexp it's unable to locate it. I 've used your sample code inside sass file

```scss
/*
---
name: Button
category: Components
---

Buttons can be used with `<a>`, `<button>`, and `<input>` elements.

Types of buttons:
- Default: Standard button
...
```

The regexp seem's okay on some online validators, so I'm thinking there is some sort of error in the string format.

To illustrate you, the real problem is here:

```js
// Extracts blocks from description
// here description seems a valid string with chosen content
var blocks = description.match(/```(.*\n)+?```/g);
// returns null
```

Here's my current environment:
OS: Windows 7
Node: 8.5
Yarn: 1.3.2
Stylemark: 0.5.9

**Comment by franklintarter** [comment]
@aitorllj93 

I had this same issue that came up when I did a clean reinstall on my system. As much as I love Visual Studio Code, it's default line break is CRLF.. Come on! Changing your line breaks to LF should resolve.

**Comment by mpetrovich** [comment]
Fixed in v0.6.6

Please reopen if you still encounter issues.
